Assets/Scripts/Layers/LayerTypes/HierarchicalObjectLayerGameObject.cs

```diff
@@ -21,17 +21,24 @@ public class HierarchicalObjectLayerGameObject : LayerGameObject, IPointerClickH
         private Quaternion previousRotation;
         private Vector3 previousScale;
 
+        protected Coordinate coord;
```

Comment:
Ik weet niet in hoeverre dit netjes moet aangezien het een POC is, maar het is voor mij iets netter om transformPropertyData.Coordinate te gebruiken